queue_services/entity-filer/src/entity_filer/filing_processors/dissolution.py

```diff
@@ -18,24 +18,28 @@
 import dpath
 import sentry_sdk
 from entity_queue_common.service_utils import QueueException, logger
-from legal_api.models import Business, Filing
+from legal_api.models import Business, Document, Filing
+from legal_api.models.document import DocumentType
+from legal_api.services.minio import MinioService
 from legal_api.utils.datetime import datetime
 
 from entity_filer.filing_meta import FilingMeta
 from entity_filer.filing_processors.filing_components import create_office, filings
 from entity_filer.filing_processors.filing_components.parties import update_parties
+from entity_filer.utils import replace_file_with_certified_copy
 
 
-def process(business: Business, filing: Dict, filing_meta: FilingMeta):
+def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: FilingMeta):
     """Render the dissolution filing unto the model objects."""
     if not (dissolution_filing := filing.get('dissolution')):
         logger.error('Could not find Dissolution in: %s', filing)
         raise QueueException(f'legal_filing:Dissolution missing from {filing}')
 
     logger.debug('processing dissolution: %s', filing)
     dissolution_date = datetime.fromisoformat(dissolution_filing.get('dissolutionDate'))
-    # Currently we don't use this for anything?
+    # FUTURE: remove hasLiabilities from schema as it can be derived from dissolutionStatementType
     # has_liabilities = filing['dissolution'].get('hasLiabilities')
+
     business.dissolution_date = dissolution_date
 
     # remove all directors and add custodial party if in filing
```

Comment:
Could you do me favour and change this comment, directors are not removed and the custodian is added.
It's different than the other party updates.